macosx/Controller.m

```diff
@@ -904,7 +904,8 @@ - (void) download: (NSURLDownload *) download decideDestinationWithSuggestedFile
 
 -(void) download: (NSURLDownload *) download didCreateDestination: (NSString *) path
 {
-    [(NSMutableDictionary *)[fPendingTorrentDownloads objectForKey: [[download request] URL]] setObject: path forKey: @"Path"];
+    NSMutableDictionary *dict = (NSMutableDictionary *)fPendingTorrentDownloads[[[download request] URL]];
```

Comment:
It would be great if there was a guide on the project wiki outlining the general coding and formatting style. Are there any plans for such a document?